tests/io/test_read_eea_aqerep_v2.py

```diff
@@ -42,14 +51,15 @@ def test_read(reader):
         assert isinstance(data, UngriddedData)
 
         for stat_idx, statid in enumerate(station_id[var_name]):
+            print(statid)
```

Comment:
`stdout` and `stderr` are captured by pytest, so you will not see this print unless the test fails, in witch case the print will bet lost between the stack trace.

Reply:
leftover from testing; removed